src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -109,6 +109,8 @@ components:
           example: Smith
         certificateLanguage:
           $ref: '#/components/schemas/CertificateLanguage'
+        supportingInformationFormat:
```

Comment:
Can we add this new property to the `required` array please?
We need it to be a mandatory property in the message (though see my other comment about chicken and egg 😁 )

Reply:
@NathanRussellValtech , maybe for this change I only change the Jackson Configuration to allow unknown fields and then update VCA message and the spec to include the new field and then update `print-api` with the new spec as well.